cumulusci/robotframework/tests/salesforce/api.robot

```diff
@@ -101,22 +101,13 @@ Collection API Errors Test
         ...  FirstName=User {{number}}
         ...  LastName={{fake.last_name}}
         ...  Xyzzy=qwertz
-    Run Keyword And Expect Error   SalesforceMalformedRequest*   Salesforce Collection Insert  ${objects}
+    Run Keyword And Expect Error   *Xyzzy*   Salesforce Collection Insert  ${objects}
 
     @{objects} =  Generate Test Data  Contact  20
         ...  FirstName=User {{number}}
         ...  LastName=
     Run Keyword And Expect Error   Error*  Salesforce Collection Insert  ${objects}
 
-    @{objects} =  Generate Test Data  Contact  20
```

Comment:
The platform semantics seem to changed so that this no longer throws an exception. There is nothing to test, really, anymore. We can't test our exception handling because there is no exception and we shouldn't test that NO exception is thrown because we don't really care if they change it back to start throwing exceptions again in the future.